library/CoreData/Base/iCloudCDStack.swift

```diff
@@ -174,6 +174,8 @@ public class iCloudCDStack: DefaultCDStack
             self!.mainContext = self!.createMainContext(self!.rootSavingContext)
             self!.addObservers()
             self!.stackInitialized = true
+            
+            NSNotificationCenter.defaultCenter().postNotificationName("iCloudStackInitialized", object: nil)
```

Comment:
If you look at the example from the previous commit, you'll see the iCloud example crash. It crashes because i'm instantiating the iCloudStack and then using it before the initialize() is finished since it is ran asynchronously.